src/views/clusteroverview/SettingsTab/UserTab/components/ManageSSHKeySection/ProjectSSHKeysList/ProjectSSHKeysList.tsx

```diff
@@ -39,60 +39,63 @@ const ProjectSSHKeysList: FC = () => {
 
   const loaded = useMemo(() => settingsLoaded && projectsLoaded, [settingsLoaded, projectsLoaded]);
 
-  if (!loaded) return <Skeleton />;
-
-  const addSSHKey = () => {
+  const addSSHKey = useCallback(() => {
     setSSHSecretKeys((prev) => ({ ...prev, '': '' }));
     setIsAddingKey(true);
-  };
+  }, [setIsAddingKey, setSSHSecretKeys]);
+
+  useEffect(() => {
+    addSSHKey();
+  }, [addSSHKey]);
```

Comment:
If you want initial state with one empty row, you can go to the file useAuthorizedSSHKeys.ts and add there the following:
`const [sshSecretKeys, setSSHSecretKeys] = useState<ProjectSSHSecretMap>({ '': '' });`
`const [isAddingKey, setIsAddingKey] = useState(true);`
and change the useEffect in that file to 
```
useEffect(() => {
    if (loaded && !isEmpty(authorizedSSHKeys)) {
      setSSHSecretKeys(authorizedSSHKeys);
      setIsAddingKey(false);
    }
  }, [loaded, authorizedSSHKeys]);
```
this should solve  the issue in the screencast I've added